src/components/Page/components/Header/Header.tsx

```diff
@@ -31,143 +29,92 @@ export interface HeaderProps extends TitleProps {
   actionGroups?: MenuGroupDescriptor[];
 }
 
-interface State {
-  mobileView?: boolean;
-}
+export function Header({
```

Comment:
After removing the breakpoint logic I was able to convert Header into a functional component 😄